### DISCUSSION TITLE
Font Awesome 6.0 support

### DISCUSSION THREAD

**Discussion by hawkeye64** [quasarframework/quasar#11329] [Ideas / Proposals]
Using this discussion as a placeholder.

Currently `Font Awesome 6.0` is in beta. When it is no longer in beta we can put v5, in extras, in retirement, and deal with v6.0. We can't jump to the beta because we cannot be sure v5 will have any updates before the v6 release.

In the meantime, it looks like v6 has additional prefixes which will need to be supported in QIcon/extras (disqualifying the duotone):

```html
<i class="fa-solid fa-alarm-clock"></i>
<i class="fa-regular fa-alarm-clock"></i>
<i class="fa-light fa-alarm-clock"></i>
<i class="fa-thin fa-alarm-clock"></i>
```

The current npm install is:

```
fontawesome-free@next
```

**Comment by disarticulate** [comment]
It's no longer in beta I believe.

**Comment by hawkeye64** [comment]
There are PRs and will likely be in the next release. https://github.com/quasarframework/quasar/pull/12533
I see there are now some conflicts, so I will work on that soon.
